### PR TITLE
Switch all descriptions from .NET Core to .NET

### PR DESCRIPTION
**Pull request by KGonzalezASC**
The commits are staged in 3 pushes so it is possible to cherry-pick was it an acceptable changed.

Here are all changes:

-About Delegates.cs: .Net → .NET

-About Exceptions.cs: executing and something were misspelled.

-About Float.cs: Changes all references from .NET Core to just .NET

-About Generic Containers.cs: .Net →.NET

-.tours were updated

-Attempted to updated README.md picture name and updated contents that I thought was safe to change. Any links existing with reference to old (dotnetcorekoans) naming scheme were left unchanged other than that.

-Also fixed the link for: "[Learn C# on Windows, OSX or Linux with the .NET Core Koans]"in the README.md which currently routes to 404

Please let me know if the are any issues with this commit and anything I can improve upon :nerd_face:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This project is based on the work of [Cory Foy](https://github.com/CoryFoy) and
```
Comment (MaxAtoms) [inline README.md:57]: > If you are interested in learning VB.NET or the Full .NET Framework (windows only), please look at his fine work.

I suggest we change this sentence to:

> If you are interested in learning VB.NET, please look at his fine work.

The .NET Framework Koans do not address legacy .NET Framework functionality, as far as I can tell. Additionally, .NET has replaced the old .NET Framework.

File: DotNetKoans.csproj [hunk 2]
```diff
@@ -4,14 +4,14 @@
     <VersionPrefix>3.1.0</VersionPrefix>
```
Comment (MaxAtoms) [inline DotNetKoans.csproj:4]: You missed the `Core` in the `<Description>` 😉

**Review by MaxAtoms — CHANGES_REQUESTED** [review]
Just some minor suggestions/additions 🙂

File: .vscode/launch.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": ".NET Core Launch (console)",
+            "name": ".NET Launch (console)",
```
[the remark this answers is not in the document]
Comment (KGonzalezASC) [reply .vscode/launch.json:5]: Would I also need to change "program" value to match the naming scheme?

File: DotNetKoans.csproj [hunk 4]
```diff
@@ -4,14 +4,14 @@
...
     <TargetFramework>net6.0</TargetFramework>
     <NoWarn>$(NoWarn);CS0649;CS0162;CS0168;CS0184;CS0219;xUnit2015;xUnit2013</NoWarn>
-    <AssemblyName>DotNetCoreKoans</AssemblyName>
```
[the remark this answers is not in the document]
Comment (KGonzalezASC) [reply DotNetKoans.csproj:7]: I think it was deleted by accident, sorry about that

File: .all-contributorsrc [hunk 5]
```diff
@@ -346,7 +346,7 @@
     }
   ],
   "contributorsPerLine": 7,
-  "projectName": "DotNetCoreKoans",
+  "projectName": "DotNetKoans",
```
[the remark this answers is not in the document]
Comment (KGonzalezASC) [reply .all-contributorsrc:349]: I think keeping it the same is best if the change is uncertain. I resolved and commit all the changes suggested let me know if there anything that need to be done on my end!

File: .vscode/launch.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (MaxAtoms) [reply .vscode/launch.json:5]: Yes, that needs to be changed as well.
I have noted this in this separate comment below: https://github.com/NotMyself/DotNetCoreKoans/pull/184#discussion_r1161263897

[the remark this answers is not in the document]
Comment (MaxAtoms) [reply .vscode/launch.json:5]: I changed the program path myself. Thanks for giving me write access to your branch that was very helpful 🙂

**Review by MaxAtoms — APPROVED** [review]

**Comment by MaxAtoms** [comment]
https://github.com/all-contributors please add @KGonzalezASC for code

**Comment by MaxAtoms** [comment]
@all-contributors please add @KGonzalezASC for code

File: .all-contributorsrc (hunk 5 above)
[the remark this answers is not in the document]
Comment (MaxAtoms) [reply .all-contributorsrc:349]: The bot actually reverts the change by itself: https://github.com/NotMyself/DotNetCoreKoans/blob/e0d70b7890ce444682738d5d052a43240ef5cb3a/.all-contributorsrc#L358

**Pull request merged**